### ISSUE TITLE
Why do staff get high level access to "sensitive" data now

### ISSUE THREAD

**Issue by JasonP01** [xpdustry/imperium#14]
I personally am curious as to why you are giving us (staff) access to uuids/ip when thats never been the case on cn

My understanding is that not even log trusted admins that much to give them access to that info, (plus security risk for compromised accounts)

I just want to hear your reasoning for giving us these, feels... not good to have (*in no way am i insinuating i will do harm with these ~~again~~*)

Possible reason im thinking is too lazy to remake the trace menu that we have (doesnt explain the discord part tho)

**Comment by JasonP01** [comment]
> My understanding is that not even log trusted admins that much to give them access to that info

And he has stated that he only gave admin to people he personally trusted

**Comment by JasonP01** [comment]
Wish i could label my issues myself 😔

**Comment by phinner** [comment]
> I personally am curious as to why you are giving us (staff) access to uuids/ip when thats never been the case on cn

:raised_eyebrow: I think you get this feeling due to the fact I promoted every overseer to mod.
And it happends mods have access to uuid/ip. Not overseers.

> My understanding is that not even log trusted admins that much to give them access to that info, (plus security risk for compromised accounts)

:skull: I do understand the paranoïa.

Ok, in short, the new system is:
- OVERSEER: Helper players with kick permissions and stuff. (without sensitive player info).
- MOD: Official moderator with kick and other punishment related stuff (pardon and stuff). (can see sensitive info).
- ADMIN: All the above but with permissions that allows them to directly modify the servers, such as managing maps for example.

Tell me if I got it right.

**Comment by JasonP01** [comment]
Seems right, just i dont see a reason why mods need uuid/ip, more specifically on the ip part, its just a risk that i dont think we should have.

 i get uuid a lil more, its hard to change your uuid as a regular user in the game as is (if u dont know coding that well), and alot servers that i know needs ip to be correct to get staff ranks ingame (for if they do ip spoof), im not saying our staff will be... evil... but its a possibility that i acount for myself 
I do like that we get uuids ingame currently so that trace tells us if a user changed their name (most recent case of this was sandbox where someone griefed, left, changed their name and rejoined to play it off), and well we dont get to see if they changed their name (automatically) currently

So basically, to sumarise i think having mods get ip is not smort, admins are more suited to that with their more managerial style with the servers itself, uuids are fine to have, only smart people can abuse it.

(These are basically meant to be 1 wall of text when read, they are seperated for my and others readability, hate having to decipher a wall of text)

**Comment by phinner** [comment]
> So basically, to sumarise i think having mods get ip is not smort, admins are more suited to that with their more managerial style with the servers itself, uuids are fine to have, only smart people can abuse it.

Ok, I get it now, I will remove uuid/ip visibility from mods and only show user id instead of uuid.

> (These are basically meant to be 1 wall of text when read, they are seperated for my and others readability, hate having to decipher a wall of text)

The feeling is mutual.
